asterisk/agi/src/Agi/Action/FriendCallAction.php

```diff
@@ -66,6 +66,13 @@ public function process()
         // Some verbose dolan pls
         $this->agi->notice("Preparing call to %s through friend <cyan>%s</cyan>", $number, $friend);
 
+        // Intervpbx friends can only call to valid extensions in destination company
+        if ($friend->getDirectConnectivity() === FriendInterface::DIRECTCONNECTIVITY_INTERVPBX &&
```

Comment:
Could we name conditions here? Set them into a variable or  add a couple of methods to Friend as well